packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilter.ts

```diff
@@ -374,7 +374,14 @@ function parseGenericFilterRule(rule: GenericInstanceFilterRule, ctx: GenericFil
   return {
     field,
     operator: rule.operator,
-    value: rule.value ? { valueFormat: PropertyValueFormat.Primitive, displayValue: rule.value.displayValue, value: rule.value.rawValue } : undefined,
+    value: rule.value
+      ? {
+          valueFormat: PropertyValueFormat.Primitive,
+          displayValue: rule.value.displayValue,
+          value: rule.value.rawValue,
+          ...(rule.value && "roundingError" in rule.value ? { roundingError: rule.value.roundingError } : undefined),
```

Comment:
do we need the `rule.value` check?

```suggestion
          ...("roundingError" in rule.value ? { roundingError: rule.value.roundingError } : undefined),
```